src/pages/Community/CommunityForm.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import * as C from './CommunityStyle';
+import { IoMdArrowDropdown } from 'react-icons/io';
+import CommunityHashTag from './CommunityHashTag';
+
+export default function CommunityForm({ findedItem }) {
+  const [categoryBox, setCategoryBox] = useState();
+  const [categoryList, setCategoryList] = useState('CATEGORY');
+  const [hashTagItems, setHashTagItems] = useState([]);
+  const [content, setContent] = useState('');
+  const [hashTag, setHashTag] = useState(['#']);
+  const handleHashTag = e => {
+    e.preventDefault();
+    setHashTag(e.target.value);
+  };
+  const hadlecontent = e => {
+    setContent(e.target.value);
+    findedItem.content = e.target.value;
+  };
+
+  const addHashTag = e => {
+    e.preventDefault();
+    if (hashTagItems.length > 5) {
+      return;
+    }
+    if (e.key === 'Enter' && e.nativeEvent.isComposing === false) {
+      setHashTagItems([hashTag, ...hashTagItems]);
+      setHashTag('#');
+      findedItem.hashTag = [hashTag, ...findedItem.hashTag];
+    }
+  };
+  const selected = () => {
+    setCategoryBox(!categoryBox);
+  };
+  const changeSelect = e => {
+    const { innerText } = e.target;
+    setCategoryList(innerText);
+
+    findedItem.category = e.target.innerText;
+  };
```

Comment:
```suggestion
  const changeSelect = e => {
    setCategoryList(e.target.innerText);
    findedItem.category = e.target.innerText;
  };
```
하나의 값이라면 구조분해할당 안 하셔도 좋을 것 같습니다!